### PR TITLE
Feature/offset in axis

### PR DESCRIPTION
**Pull request by Brynjulf**
#68

### CODE REVIEW

File: src/components/axis.ts [hunk 1]
```diff
@@ -1,33 +1,53 @@
...
-  mainGroup: Selection<SVGElement, any, null, undefined>;
-  scaleX: ScaleLinear<number, number>;
-  scaleY: ScaleLinear<number, number>;
+  private mainGroup: Selection<SVGElement, any, null, undefined>;
+  private _scaleX: ScaleLinear<number, number>;
+  private _scaleY: ScaleLinear<number, number>;
   showLabels = true;
   labelXDesc: string;
   labelYDesc: string;
   measurement: string;
```
Comment (HavardNJ) [inline src/components/axis.ts:19]: What is measurement, should it me unit instead?

File: src/components/axis.ts [hunk 2]
```diff
@@ -96,26 +124,50 @@ export class Axis {
...
+
+  get offsetY(): number {
+    return this._offsetY;
+  }
+
+  set offsetY(offset: number) {
+    this._offsetY = offset;
+  }
+
+  get scaleX(): ScaleLinear<number, number> {
```
Comment (HavardNJ) [inline src/components/axis.ts:166]: Is this function used/neccesary? If it is, does it need to copy the scale as the code is now?

File: src/components/axis.ts (hunk 1 above)
Comment (Brynjulf) [reply src/components/axis.ts:19]: sure, or unitOfMeasure

File: src/components/axis.ts [hunk 3]
```diff
@@ -96,26 +124,50 @@ export class Axis {
...
+    return this._scaleX.copy();
+  }
+
+  get scaleY(): ScaleLinear<number, number> {
```
Comment (HavardNJ) [inline src/components/axis.ts:170]: Is this function used/neccesary? If it is, does it need to copy the scale as the code is now?

File: src/components/axis.ts (hunk 2 above)
Comment (Brynjulf) [reply src/components/axis.ts:166]: it is not used as far as i know, but if anyone needs to know what domain/range the axis has, then we need to make sure that they are not allowed to manipulate the scale directly

**Pull request merged**